### PR TITLE
TYPO3  backend login

### PR DESCRIPTION
**Pull request by franzholz**
There is a case when no TYPO3 backend login is possible.
See TYPO3 slack typo3-cms from 1st July 2021.

### CODE REVIEW

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 1]
```diff
@@ -94,6 +94,10 @@ description of the settings carefully at least once, so you get an
 impression of what you can configure. Normally you *can*, but you *don't
 have to* change anything here during installation, as the previous steps
 took care of the most important settings.
+Pay attention to use a https website and set the BE loginSecurityLevel to normal.
```
Comment (mbrodala) [inline Documentation/In-depth/TheInstallTool/Index.rst:97]:
```suggestion
Pay attention to use HTTPS for your website and set :code:`$GLOBALS['TYPO3_CONF_VARS']['BE']['loginSecurityLevel']` to `normal`.
```

This is not exactly true. Even with HTTPS you can use RSAAuth, it's just pointless. Thus we should rephrase this to a recommendation.

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 2]
```diff
@@ -94,6 +94,10 @@ description of the settings carefully at least once, so you get an
...
+On all http sites however you must set this to rsa and
```
Comment (mbrodala) [inline Documentation/In-depth/TheInstallTool/Index.rst:98]:
```suggestion
However, on all websites without HTTPS you should set this to `rsa` and 
```

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 3]
```diff
@@ -94,6 +94,10 @@ description of the settings carefully at least once, so you get an
 impression of what you can configure. Normally you *can*, but you *don't
 have to* change anything here during installation, as the previous steps
 took care of the most important settings.
+Pay attention to use a https website and set the BE loginSecurityLevel to normal.
+On all http sites however you must set this to rsa and 
+activate the system extension rsaauth. Otherwise you will not be able to log
```
Comment (mbrodala) [inline Documentation/In-depth/TheInstallTool/Index.rst:99]:
```suggestion
activate the system extension `rsaauth`. Otherwise you will not be able to log
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 4]
```diff
@@ -94,6 +94,10 @@ description of the settings carefully at least once, so you get an
 impression of what you can configure. Normally you *can*, but you *don't
 have to* change anything here during installation, as the previous steps
 took care of the most important settings.
+Pay attention to use HTTPS for your website and set :code:`$GLOBALS['TYPO3_CONF_VARS']['BE']['loginSecurityLevel']` to `normal`.
+However, on all websites without HTTPS you should set this to `rsa` and 
+activate the system extension `rsaauth`. Otherwise you will not be able to log
```
Comment (josefglatz) [inline Documentation/In-depth/TheInstallTool/Index.rst:99]: Should we also mention that using that this encryption approach is deprecated and is not part of the TYPO3 core anymore? https://github.com/FriendsOfTYPO3/rsaauth#current-state

**Review by josefglatz — COMMENTED** [review]
Generally good content! Except that we should avoid motivating people using rsaauth.

File: Documentation/In-depth/TheInstallTool/Index.rst (hunk 4 above)
Comment (mbrodala) [reply Documentation/In-depth/TheInstallTool/Index.rst:99]: Yes, good idea.

Comment (susannemoog) [reply Documentation/In-depth/TheInstallTool/Index.rst:99]: I wouldn't add that in the 8.7 docs- that's part of the upgrade process I think.

Comment (mbrodala) [reply Documentation/In-depth/TheInstallTool/Index.rst:99]: Indeed, I forgot the v8 context. This indeed only applies to v9+.

**Review by mbrodala — APPROVED** [review]
